clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/view/KubernetesClusterProvider.groovy

```diff
@@ -148,6 +147,15 @@ class KubernetesClusterProvider implements ClusterProvider<KubernetesCluster> {
       serverGroup.loadBalancers?.each {
         serverGroup.securityGroups.addAll(securityGroups[it])
       }
+
+      def imageList = new ArrayList()
+      for (def container : serverGroup.deployDescription.containers) {
+        imageList.add(container.imageDescription.repository + ":" + container.imageDescription.tag)
+      }
+      if (!imageList.isEmpty()) {
+        Map buildInfo = [images: imageList]
```

Comment:
I had done it so I could check on buildInfo.images in the UI but you are right, that if its empty then we likely have a problem.  Ill remove the isEmpty check.